packages/itmat-ui-react/src/components/lxd/util/formatUtils.ts

```diff
@@ -0,0 +1,43 @@
+// src/utils/formatUtils.ts
+import { Cpu, Memory, Storage, Gpu, Network, Pci } from '@itmat-broker/itmat-types';
+
+// Format CPU information
+export const formatCPUInfo = (cpu: Cpu) => {
+    if (!cpu.sockets) return 'CPU information not available';
+    const coreCount = cpu.sockets.reduce((acc, socket) => acc + (socket.cores ? socket.cores.length : 0), 0);
+    return `CPU: ${cpu.total} Socket(s), ${coreCount} Core(s) Total, Architecture: ${cpu.architecture}`;
+};
+
+// Format Memory information
+export const formatMemoryInfo = (memory: Memory) => {
+    return `Total Memory: ${formatBytes(memory.total)}, Used: ${formatBytes(memory.used)}, Free: ${formatBytes(memory.total - memory.used)}`;
+};
+
+// Format Storage information
+export const formatStorageInfo = (storage: Storage) => {
+    if (!storage.disks) return 'Storage information not available';
+    const totalSize = storage.disks.reduce((acc, disk) => acc + disk.size, 0);
+    return `Storage: ${storage.total} Disk(s), Total Size: ${formatBytes(totalSize)}`;
+};
+
+export const formatGPUInfo = (gpu: Gpu) => {
+    return `GPU: ${gpu.total} Card(s)`;
+};
+
+export const formatNetworkInfo = (network: Network) => {
+    return `Network: ${network.total} Card(s)`;
+};
+
+export const formatPCIInfo = (pci: Pci) => {
+    return `PCI: ${pci.total} Device(s)`;
+};
```

Comment:
As described in an earlier comment, this may not be required to be known by the end-user.